spec/main.spec.ts

```diff
@@ -74,22 +82,36 @@ describe('main', () => {
       expect(context.timestamp).to.equal('2018-03-28T18:58:50.370Z');
     });
 
-    it('should generate auth and authType for database functions', () => {
-      const context = wrap(constructCF('google.firebase.database.ref.write'))(
-        'data'
-      ).context;
-      expect(context.auth).to.equal(null);
-      expect(context.authType).to.equal('UNAUTHENTICATED');
-    });
+    context('database functions', () => {
```

Comment:
This seems to be a refactor without a benefit, unless I"m missing something. We also just use `describe` in our codebases instead of `context`. We also generally use `beforeEach` over `before` because the nanosecond we save by not calling a setup function repeatedly is less important than hairpulling if a test changes some of the state between runs.